lib/src/rules/avoid_void_async.dart

```diff
@@ -29,6 +29,19 @@ Future<void> f() async {}
 Future<void> f2() async => null;
 ```
 
+**EXCEPTION**
+
+An exception is made for `main`, where the `Future` annotation *can* (and
+generally should) be dropped in favor of `void`.
```

Comment:
Maybe somehow clarify this only applies to top-level `main` functions?

```suggestion
An exception is made for top-level `main` functions, 
where the `Future` annotation *can* (and generally should) 
be dropped in favor of `void`.
```

Reply:
Good catch.  Will do!